lib/utils/report.mjs

```diff
@@ -115,6 +125,45 @@ export default function report(problem) {
 	result.warn(warningMessage, warningProperties);
 }
 
+/**
+ * @param {Problem} problem
+ */
+function problemRangeDeprecations(problem) {
+	// TODO: `config.quietDeprecationWarnings` is not available in `problem`
+
+	if (!problem.node) {
```

Comment:
[suggestion] To check the case when `node` is falsy like `undefined`, does the following make sense?

```suggestion
	if ('node' in problem) {
```

Reply:
I am unsure 🤔
A value of `undefined`, `null`, ... would also be an invalid argument for our purposes.

Maybe we need to validate both:
- that the `node` argument was provided
- that the provided value is a `PostCSS` node

It is not inconceivable that a rule author might try to pass a `postcss-selector-parser` node to `utils.report()`.

Reply:
Oh, I completely misunderstood! 🙇🏼 

The current check is no problem 👍🏼